### DISCUSSION TITLE
Moving data directory

### DISCUSSION THREAD

**Discussion by geomule** [jonaswinkler/paperless-ng#759] [Support] [answered]
Hi, I would like to move the data directory to a difference partition (ext 4 mounted with the `noexec` option). Would it be as simple as stopping all Paperless services, moving the data directory, changing the entries in the `paperless.conf` file and then firing up all the services? This is driven by the fact that the partition where I have the media folder (seems to be the biggest consumer of space) is saved in `/opt/paperless-ng/media` folder and I would like  it on the `/mnt/big_partition/paperless-ng/media`. Do you think I'll break the system if I do the above?

The actual database is not a problem as I'm using Postgres although if Paperless is storing file paths in Postgress then I can see this not being an option and I would need to start messing around with database table (or am I wrong?).

Thanks

**Comment by jonaswinkler** [comment, marked as answer]
Hello!

* the data directory is what contains some auxiliary data, such as the search index, data for the automatic classifier, and so on.
* the media directory contains your actual documents.

You can shut down paperless, move these around, update the configuration, and start paperless again. The database contains relative paths, so no issues with moving directories here.

**Reply by jonaswinkler** [reply]
You can also use the sanity checker (search the documentation) before and after the operation to make sure you've got it right.

**Reply by geomule** [reply]
Thank you very much and also for the great effort in creating this great piece of software that seems quite well thought through.

Steps I followed for others that come across this:

1) Stop all paperless services with `systemctl`
2) `cp -R /opt/paperless-ng /mnt/bigpartition/paperless/media`
3) Change the `PAPERLESS_MEDIA_ROOT` setting to `PAPERLESS_MEDIA_ROOT=/mnt/bigpartition/paperless/media`
4) Restart all the Paperless services with `sudo systemctl start paperless-webserver`, etc.
